src/main/antlr/at/ac/tuwien/kr/alpha/antlr/ASPCore2.g4

```diff
@@ -42,12 +42,14 @@ weight_at_level : term (AT term)? (COMMA terms)?;
 
 naf_literals : naf_literal (COMMA naf_literals)?;
 
-naf_literal : NAF? (classical_literal | builtin_atom);
+naf_literal : NAF? (external_atom | classical_literal | builtin_atom);
 
 classical_literal : MINUS? ID (PAREN_OPEN terms PAREN_CLOSE)?;
 
 builtin_atom : term binop term;
 
+external_atom : MINUS? AMPERSAND ID (SQUARE_OPEN input = terms SQUARE_CLOSE)? (PAREN_OPEN output = terms PAREN_CLOSE)?;
```

Comment:
The first parentheses for input cannot be optional, as `&ext(foo)` is ambigous, i.e., `foo` could be input or output. I would also drop the optional MINUS -- this is not supported anyway.

Reply:
I'd agree if input and output list would use the same parentheses, though output uses the `[]` pair while input uses `()`, inspired by [DLV-HEX (Section 2)](http://www.kr.tuwien.ac.at/staff/eiter/et-archive/iswc05-dlvhex.pdf).

Will remove `MINUS?`.

Reply:
Yes, the `MINUS?` probably is not needed. Sorry, I twice saw the wrong parentheses.